Assets/PlayerAssets/Scripts/Character.cs

```diff
@@ -0,0 +1,64 @@
+﻿using UnityEngine;
+using ARPGTD.CharacterStats;
+using UnityEngine.AI;
+using System.Collections.Generic;
+
+public class Character : MonoBehaviour {
+
+    NavMeshAgent pc;
+    public CharacterStat Strength;
+    public CharacterStat Agility;
+    public CharacterStat Intillect;
+    public CharacterStat Stamina;
+
+    float moveSpeed;
+    public Item[] inventory;
+
+    void Start(){
+        Strength.BaseValue = 10;
+        Agility.BaseValue = 10;
+        Intillect.BaseValue = 10;
+        Stamina.BaseValue = 10;
+        moveSpeed = 100;
+
+        inventory = new Item[] {null,null,null,null,null,null};
+
+        pc = transform.GetComponent<NavMeshAgent>();
+        pc.speed = moveSpeed;
+    }
+
+    public void AddItemToInventory(Item item){
+        for(int i=0; i<6; i++){
```

Comment:
What's this magic number? try to avoid random numbers like this. Also, you said things are going to be changed right?